Automattic/newspack-sponsors
# fix: update ci images and NPM packages

### All Submissions:






### Changes proposed in this Pull Request:



As seen on https://github.com/Automattic/newspack-ads/pull/214, recent changes made to `circleci/php` resulted in our builds fail due to an incompatibility with our `node-sass` dependency.

[`circleci/` images are being deprecated and will lose support starting December 2021](https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034). We should start moving towards maintained images.

As with the other Newspack repositories that use CircleCI, this PR implements the use of `cimg/` images and updates some of the build dependencies.

This PR also updates NPM packages to bring dependencies up-to-date with the latest Calypso dependencies. This means the required/supported NPM version is now v16.11.1.

### How to test the changes in this Pull Request:

1. Ensure that you can still run all linting and build commands (`npm run lint`, `npm run build`, `npm run dev`).
2. Ensure that all CI jobs pass. 

### Other information:
